README.md

```diff
@@ -35,6 +35,26 @@ From there, you can use one of our technology integrations such those for
 Have fun! We have a Google Group and a Slack channel we use for both developers and 
 users. Check them out [here](https://projectnessie.org/develop/).
 
+### Authentication
+
+By default, Nessie servers run with authentication disabled and all requests are processed under the "anonymous"
+user identity.
+
+Nessie supports bearer tokens and uses [OpenID Connect](https://openid.net/connect/) for validating them.
```

Comment:
nit: just mention something like "and HTTP BASIC for test purposes" or so.

Reply:
I'm not sure. Prebuilt Nessie jars and docker images actually do not have Basic Auth enabled. Basic auth works only in unit tests (when the `test` quarkus` profile is active)... but it is worth putting in README?

Reply:
Ah, true, you‘re right